sparkle/Components/Sparkle-latex-source/SparkleReport.bib

```diff
@@ -100,3 +91,22 @@ @article{FawcettHoos16
   biburl    = {https://dblp.org/rec/journals/heuristics/FawcettH16.bib},
   bibsource = {dblp computer science bibliography, https://dblp.org}
 }
+
+@inproceedings{SMAC,
```

Comment:
I would adhere to the AIM citation key convention as was used previously

Reply:
This way we can easily switch citation by name of the configurator, otherwise we would need to specify another variable translation in the report generator which is possible of course. Do you think that is worth it?